### PR TITLE
Feature/filerBarUpdate

### PR DESCRIPTION
**Pull request by RomchikSt**
Trello: https://trello.com/c/UndJeK9C/17-filter-bar-ui-update 

I have a problem with the URL (I have shown it in the video)
Also cleaned my code 
After:
https://github.com/unreal998/top-kolesa-fe/assets/140477189/1b182c9b-1e06-4f6d-ae7b-b966c1eaf257

### CODE REVIEW

File: src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx [hunk 1]
```diff
@@ -149,17 +144,30 @@ function FilterShortMenuColumn({
...
           </Box>
-          <p
+          <Typography
+            variant="body1"
+            fontFamily={FONTS.BOLD_TEXT_FAMILY}
+            fontWeight={600}
             style={{
-              marginLeft: "11px",
+              marginLeft: `${
+                filterName === "Season"
```
Comment (unreal998) [inline src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx]: filterName === “Brand” ? “12px” : “16px”

File: src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx [hunk 2]
```diff
@@ -93,10 +86,12 @@ function FilterShortMenuColumn({
   onClick,
 }: FilterShortMenuColumnProps) {
   const dispatch = useDispatch();
+  const { t } = useTranslation();
   const visableParams = params.length > 0;
 
   function handleMenuToggle() {
-    const tabIndex = filterName === "Season" ? 4 : 5;
+    const tabIndex =
+      filterName === "Season" ? 4 : filterName === "Brand" ? 5 : 6;
```
Comment (unreal998) [inline src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx]: Don’t use a ternary operator for the complex expression

**Review by unreal998 — APPROVED** [review]
See my comment

File: src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx (hunk 1 above)
Comment (RomchikSt) [reply src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx]: done

File: src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx (hunk 2 above)
Comment (RomchikSt) [reply src/modules/shopPage/components/FilterMenu/FilterShortMenu/FilterShortMenuColumn.tsx]: replaced to "switch"

**Pull request merged**